src/covjson_pydantic/base_models.py

```diff
@@ -1,32 +1,16 @@
-import orjson
 from pydantic import BaseModel as PydanticBaseModel
-from pydantic import Extra
-
-
-def orjson_dumps(v, *, default, indent=None, sort_keys=False):
-    options = orjson.OPT_NON_STR_KEYS | orjson.OPT_UTC_Z | orjson.OPT_NAIVE_UTC
-    if indent:
-        options |= orjson.OPT_INDENT_2
-
-    if sort_keys:
-        options |= orjson.OPT_SORT_KEYS
-
-    # orjson.dumps returns bytes, to match standard json.dumps we need to decode
-    return orjson.dumps(v, default=default, option=options).decode()
+from pydantic import ConfigDict
 
 
 class BaseModel(PydanticBaseModel):
-    class Config:
-        anystr_strip_whitespace = True
-        min_anystr_length = 1
-        extra = Extra.forbid
-        validate_all = True
-        validate_assignment = True
-
-        json_loads = orjson.loads
-        json_dumps = orjson_dumps
+    model_config = ConfigDict(
+        str_strip_whitespace=True,
+        str_min_length=1,
+        extra="forbid",
+        validate_default=True,
+        validate_assignment=True,
+    )
 
 
 class CovJsonBaseModel(BaseModel):
-    class Config:
-        extra = Extra.allow  # allow custom members
+    model_config = ConfigDict(extra="allow")
```

Comment:
Previously, this kept all the other config setting in BaseModel, only overwriting the `extra` setting. Now I am not so sure...

Reply:
No.. it was not needed (anymore). Changed.